app/models/order.rb

```diff
@@ -2,6 +2,8 @@ class Order < ApplicationRecord
   include OwnerField
   acts_as_tenant(:tenant)
 
+  default_scope { order(:created_at => :desc) }
+
   has_many :order_items
```

Comment:
I wonder if we should add a spec at the `spec/requests` level to validate ordering and to prove we're not going to run into an issue with the filtering code.  I think we should also build this PR off of the filtering PR to show the dependency on that PR before merging this one.

Reply:
Ok I like that idea, I rebased on my filtering PR and added some tests under the requests orders spec to validate that it's returning them in the right order and filtering is working properly.